tests/Fixer/Comment/HeaderCommentFixerTest.php

```diff
@@ -548,6 +548,73 @@ class Foo {}',
                 '<?php
 declare(strict_types=1) ?>',
             ],
+            [
+                [
+                    'header' => 'tmp',
+                    'location' => 'after_declare_strict',
+                ],
+                '#!/usr/bin/env php
```

Comment:
let's add some cases with `#!/usr/bin/php` and `#!/usr/bin/env php7.4`

Reply:
let's decide in https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/6233 what do we actually want monolithic code to be